pkg/datapath/linux/ipsec/ipsec_linux_test.go

```diff
@@ -39,11 +39,14 @@ func setupIPSecSuitePrivileged(tb testing.TB) *slog.Logger {
 }
 
 var (
-	path           = "ipsec_keys_test"
-	keysDat        = []byte("1 hmac(sha256) 0123456789abcdef0123456789abcdef cbc(aes) 0123456789abcdef0123456789abcdef\n1 hmac(sha256) 0123456789abcdef0123456789abcdef cbc(aes) 0123456789abcdef0123456789abcdef\n1 digest_null \"\" cipher_null \"\"\n")
-	keysAeadDat    = []byte("6 rfc4106(gcm(aes)) 44434241343332312423222114131211f4f3f2f1 128\n")
-	keysAeadDat256 = []byte("6 rfc4106(gcm(aes)) 44434241343332312423222114131211f4f3f2f144434241343332312423222114131211 128\n")
-	invalidKeysDat = []byte("1 test abcdefghijklmnopqrstuvwzyzABCDEF test abcdefghijklmnopqrstuvwzyzABCDEF\n")
+	path             = "ipsec_keys_test"
+	keyDat           = []byte("1 hmac(sha256) 0123456789abcdef0123456789abcdef cbc(aes) 0123456789abcdef0123456789abcdef\n")
+	keyNullDat       = []byte("2 digest_null \"\" cipher_null \"\"\n")
+	keyAeadDat       = []byte("3 rfc4106(gcm(aes)) 44434241343332312423222114131211f4f3f2f1 128\n")
+	keyAead256Dat    = []byte("4 rfc4106(gcm(aes)) 44434241343332312423222114131211f4f3f2f144434241343332312423222114131211 128\n")
+	keyInvalidDat    = []byte("1 test abcdefghijklmnopqrstuvwzyzABCDEF test abcdefghijklmnopqrstuvwzyzABCDEF\n")
+	keysSameSpiDat   = []byte("1 hmac(sha256) 0123456789abcdef0123456789abcdef cbc(aes) 0123456789abcdef0123456789abcdef\n1 digest_null \"\" cipher_null \"\"\n")
+	keysDuplicateDat = []byte("6 rfc4106(gcm(aes)) 44434241343332312423222114131211f4f3f2f1 128\n6 rfc4106(gcm(aes)) 44434241343332312423222114131211f4f3f2f1 128\n")
```

Comment:
Should we keep this exactly the same as `keysSameSpiDat`, just with different SPI and key? If the goal is to check that we mandate a new SPI, then let's test that and nothing else.